src/Integration.Vsix/Commands/HelpMenu/AboutDialog.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<platformUi:DialogWindow x:Class="SonarLint.VisualStudio.Integration.Vsix.Commands.AboutDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    Title="About SonarLint"
+    ResizeMode="NoResize"
+    SizeToContent="WidthAndHeight"
+    Background="WhiteSmoke">
```

Comment:
Btw, looks like it also has an Icon property and that should set the icon https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ee696570(v=vs.120)

Reply:
Yea I tried and it didnt work, no idea why

Reply:
Not a big deal imo